scapy/contrib/ospf.py

```diff
@@ -53,12 +53,20 @@ def __init__(self, name="options", default=0, size=8,
                5: "LSAck"}
 
 
+class _NoLLSLenField(LenField):
```

Comment:
Please add a doctring.

Reply:
This is specific to `ospf`, so I did not see any reason to share it with others